src/WebJobs.Script/Rpc/Configuration/WorkerConfigFactory.cs

```diff
@@ -31,20 +31,8 @@ public IEnumerable<WorkerConfig> GetConfigs(IEnumerable<IWorkerProvider> provide
                 var description = provider.GetDescription();
                 var languageSection = _config.GetSection($"workers:{description.Language}");
 
-                // Resolve worker path
-                // 1. If workers.{language}.path is set, use that explicitly
-                // 2. If workers.path is set, use that as the base directory + language + default path
-                // 3. Else, use the default workers directory
-
-                // get explicit worker path from config, or build relative path from default
-                var workerPath = languageSection.GetSection("path").Value;
-                if (string.IsNullOrEmpty(workerPath))
-                {
-                    var baseWorkerPath = !string.IsNullOrEmpty(_config.GetSection("workers:path").Value) ?
-                            _config.GetSection("workers:path").Value :
-                            Path.Combine(_assemblyDir, "workers");
-                    workerPath = Path.Combine(baseWorkerPath, description.Language.ToLower(), description.DefaultWorkerPath);
-                }
+                // Can override the path we load from, or we use the default path from where we loaded the config
```

Comment:
Comment about line 24 [here](https://github.com/Azure/azure-functions-host/pull/2646/files#diff-4a1578f8d75a14b4780790f365956ca4L24)... outside of the scope of your changes, but it would be nice to reuse the logic you have in the helper.